src/components/shared/PennsieveHeader/PennsieveHeader.vue

```diff
@@ -137,6 +136,7 @@ export default {
      * @returns {String}
      */
     iconSpacing() {
+      // TODO: mr-4 class is not supported, check _spacing.scss, only mr-(multiples of 8) will work according to this. Could expand or look for alternative way
```

Comment:
so mr-4 case for mobile is not working correctly? I'm not sure I'm understanding what this comment means - would you mind rephrasing it? ty!